include/bluetooth/services/fast_pair.h

```diff
@@ -20,14 +20,29 @@
 extern "C" {
 #endif
 
+/** @brief Fast Pair advertising mode. Used to generate advertsing packet.
+ */
+enum bt_fast_pair_adv_mode {
+	/** Fast Pair discoverable advertising. */
+	BT_FAST_PAIR_DISCOVERABLE_ADV,
+
+	/** Fast Pair not discoverable advertising, show UI indication. */
+	BT_FAST_PAIR_NOT_DISCOVERABLE_ADV_SHOW_UI_IND,
+
+	/** Fast Pair not discoverable advertising, hide UI indication. */
+	BT_FAST_PAIR_NOT_DISCOVERABLE_ADV_HIDE_UI_IND,
+
+	BT_FAST_PAIR_ADV_MODE_COUNT
+};
```

Comment:
For now, it seems that there will be no additional configurations. The only one that I can think of is battery notification extension during advertising, but it will be enabled by default so it won't be a part of API.